src/helm-frontend/src/utils/getVersionBaseUrl.ts

```diff
@@ -0,0 +1,7 @@
+export default function getVersionBaseUrl(): string {
+  if (window.RELEASE) {
+    return `/benchmark_output/releases/${window.RELEASE}`;
+  } else {
+    return `/benchmark_output/runs/${window.SUITE}`;
```

Comment:
This needs to change to reflect the mechanics of the incremental release system:

JSON files pertaining to aggregations across multiple runs are stored in `/benchmark_output/releases/v0.3.1/...` e.g. `/benchmark_output/releases/v0.3.1/groups.json`

JSON files pertaining to individual runs are stored in suites `/benchmark_output/runs/v0.3.0/run_name/...` - but a single release can have runs from multiple suites, so there could also be runs from `/benchmark_output/runs/v0.3.0/run_name/...`, `/benchmark_output/runs/v0.3.1/run_name/...` etc.

For each run name, you need to look at `/benchmark_output/runs/v0.3.0/runs_to_run_suites.json` file to figure out which suite each run is from ([example](https://storage.googleapis.com/crfm-helm-public/benchmark_output/releases/v0.3.0/runs_to_run_suites.json)).

For instance, in the legacy code, we load `runs_to_run_suites` [here](https://github.com/stanford-crfm/helm/blob/v0.3.0/src/helm/benchmark/static/benchmarking.js#L1662-L1667), then we compute the URLs [here](https://github.com/stanford-crfm/helm/blob/v0.3.0/src/helm/benchmark/static/benchmarking.js#L793-L817) and [here](https://github.com/stanford-crfm/helm/blob/v0.3.0/json-urls.js).

This also means that computing the URL is an async function.

In the old code, we also supported a legacy mode, where if `window.SUITE` is set but `window.RELEASE` is not set (only one of these two will be set), then _all files_ are stored in the suites e.g. `/benchmark_output/runs/v0.3.1/groups.json`, and `runs_to_run_suites.json` does not exist because all runs are in the same suite. I think we can punt on this for now... I'm not sure how much legacy stuff we want to support.